chrome/content/zotero/integration/integrationDocPrefs.xul

```diff
@@ -77,4 +84,4 @@
 		<checkbox id="storeReferences" label="&zotero.integration.prefs.storeReferences.label;"/>
 		<description class="radioDescription">&zotero.integration.prefs.storeReferences.caption;</description>
 	</vbox>
-</dialog>
\ No newline at end of file
+</dialog>
```

Comment:
I'm not entirely sure what you're asking, but I realized my suggestion would actually need a lot more explanation (interactive rebase, edit mode on that commit, `git reset HEAD^`, `git checkout -p` on that file, discarding that hunk, recommiting all changes, continuing the rebase). So probably easier to just undo this in your editor, if you can. If not, don't worry about it.